spec/features/editing_a_cma_case_spec.rb

```diff
@@ -378,10 +378,15 @@
 
 
       scenario "editing an attachment on a #{publication_state} CMA case" do
+        #TODO find a way to make this generic, regular expressions don't work in the url.
+        stub_request(:put, "#{Plek.find('asset-manager')}/assets/513a0efbed915d425e000002")
+          .with(body: %r{.*})
+          .to_return(body: asset_manager_response.to_json, status: 201)
```

Comment:
Is this what you mean? https://github.com/bblimke/webmock#matching-uris-using-regular-expressions